objects/src/testing/block.rs

```diff
@@ -586,13 +587,23 @@ impl BlockHeader {
         )
         .expect("failed to create account db");
 
-        let prev_hash = rand::rand_array().into();
-        let chain_root = chain_root.unwrap_or(rand::rand_array().into());
+        #[cfg(not(target_family = "wasm"))]
+        let (prev_hash, chain_root, nullifier_root, note_root, tx_hash, proof_hash, timestamp) = {
```

Comment:
Let's add a doc comment to the `mock()` constructor (i.e., on line 569 above) to explain what it does in general, and also mention how it works when compilation target is `wasm`.

Reply:
Added the comment